examples/toy/infra/initialize.sh

```diff
@@ -0,0 +1,69 @@
+#! /bin/bash
```

Comment:
Please refer to https://python-packaging.readthedocs.io/en/latest/non-code-files.html for packaging files other than code into the package. Probably the whole `infra/` folder should be packaged.